lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -0,0 +1,22 @@
+<a href="<%- marketingUrl %>" class="card-link">
+	<div class="banner-image">
+	</div>
+</a>
+<div class="text-section">
+	<div class="meta-info">
+		<div class="organization">
+			<% _.each(organizations, function(org){ %>
+				<span><%- gettext(org.display_name) %></span>
```

Comment:
With their own span, we can properly style them with margins and different font-weight if needed. To me the span is a helper and doesn't add much to the page if they are not yet needed.